tox.ini

```diff
@@ -47,13 +47,13 @@ commands =
     mypy: mypy --python-version 3.6 uaclient/ features/ lib/
     mypy-focal: mypy --python-version 3.7 uaclient/ features/ lib/
     black: black --check --diff uaclient/ features/ lib/ setup.py
-    behave-lxd-16.04: behave -v {posargs} --tags="series.xenial,series.lts,series.all" --tags="~upgrade"
-    behave-lxd-18.04: behave -v {posargs} --tags="series.bionic,series.lts,series.all" --tags="~upgrade"
-    behave-lxd-20.04: behave -v {posargs} --tags="series.focal,series.lts,series.all" --tags="~upgrade"
-    behave-lxd-21.04: behave -v {posargs} --tags="series.hirsute,series.all" --tags="~upgrade"
-    behave-vm-16.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.vm" --tags="series.xenial,series.all,series.lts"
-    behave-vm-18.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.vm" --tags="series.bionic,series.all,series.lts"
-    behave-vm-20.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.vm" --tags="series.focal,series.all,series.lts"
+    behave-lxd-16.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.xenial,series.lts,series.all" --tags="~upgrade"
+    behave-lxd-18.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.bionic,series.lts,series.all" --tags="~upgrade"
+    behave-lxd-20.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.focal,series.lts,series.all" --tags="~upgrade"
+    behave-lxd-21.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.hirsute,series.all" --tags="~upgrade"
+    behave-vm-16.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.vm" --tags="series.xenial,series.all,series.lts" --tags="~upgrade"
+    behave-vm-18.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.vm" --tags="series.bionic,series.all,series.lts" --tags="~upgrade"
```

Comment:
Why do we need to exclude the `upgrade` tags from the vm tests ?

Reply:
We don't - They are excluded for everything else, and since we have a specific upgrade entry, I excluded them here as well. Is there a reason to keep those?

Reply:
Since all the upgrade test are running on containers at the moment, the exclusion is unnecessary. But you right, it will not hurt us to be consistent and it on the vm tests as well

Reply:
FIPS upgrade tests are running on VMs actually